hubstorage/client.py

```diff
@@ -40,7 +40,7 @@ def _hc_retry_on_exception(err):
 class HubstorageClient(object):
 
     DEFAULT_ENDPOINT = 'https://storage.scrapinghub.com/'
-    USERAGENT = 'python-hubstorage/{0}'.format(__version__)
+    DEFAULT_USER_AGENT = 'python-hubstorage/{0}'.format(__version__)
```

Comment:
If `HubstorageClient.USERAGENT + "/c"` is not an artificial case, then `USERAGENT = None` won't work, I can set it to an empty string instead.